modules/n1ql/pages/n1ql-language-reference/createfunction.adoc

```diff
@@ -313,7 +313,13 @@ The following command registers a JavaScript function called `add` in a library
 
 [source,console]
 ----
-$ curl -v -X POST http://localhost:8093/functions/v1/libraries/math/functions/add -H 'content-type: application/json' -d '{"name": "add", "code": "function add(a, b) { let data = a + b; return data; }"}' -u Administrator:password
+$ curl -v -X POST \
+  http://localhost:8093/functions/v1/libraries/math/functions/add \
+  -u Administrator:password \
+  -H 'content-type: application/json' \
+  -d '{"name": "add",
+       "code": "function add(a, b) { let data = a + b; return data; }"
+      }'
```

Comment:
As discussed on Slack, big 👍 to reformatting this over multiple lines for readability